examples/mocha/index.html

```diff
@@ -1,26 +1,21 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
 <head>
+  <meta charset="utf-8">
   <title>Mocha</title>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-  <link rel="stylesheet" href="../../node_modules/mocha/mocha.css" />
+  <script src="../../node_modules/chai/chai.js"></script>
+  <script src="../../node_modules/mocha/mocha.js"></script>
+  <script>mocha.setup('bdd'); should=chai.should()</script>
+  <link rel="stylesheet" href="../../node_modules/mocha/mocha.css">
 </head>
-<body>
+<body onload="mocha.run()">
   <div id="mocha"></div>
-  <script src="../../node_modules/chai/chai.js" type="text/javascript"></script>
-  <script> var should = chai.should(); </script>
-  <script src="../../node_modules/mocha/mocha.js" type="text/javascript"></script>
-  <script>mocha.setup('bdd')</script>
-  <script src="./blogpost.js" type="text/javascript"></script>
-  <script src="test.js"></script>
   <script>
     function assert(expr, msg) {
       if (!expr) throw new Error(msg || 'failed');
     }
-
-    onload = function(){
-      mocha.run();
-    };
   </script>
+  <script src="blogpost.js"></script>
+  <script src="test.js"></script>
```

Comment:
Yes, I understand that, I just wanted to learn about why you made these changes. It doesn’t really look functionally different from what was there before, so I’d like to understand how your changes improve the example. 😊